src/gt4py/next/program_processors/runners/dace_fieldview/gtir_tasklet_codegen.py

```diff
@@ -0,0 +1,118 @@
+    "arcsin": "asin({})",
+    "arccos": "acos({})",
+    "arctan": "atan({})",
+    "sinh": "math.sinh({})",
+    "cosh": "math.cosh({})",
+    "tanh": "math.tanh({})",
+    "arcsinh": "asinh({})",
+    "arccosh": "acosh({})",
+    "arctanh": "atanh({})",
+    "sqrt": "math.sqrt({})",
+    "exp": "math.exp({})",
+    "log": "math.log({})",
+    "gamma": "tgamma({})",
+    "cbrt": "cbrt({})",
+    "isfinite": "isfinite({})",
+    "isinf": "isinf({})",
+    "isnan": "isnan({})",
+    "floor": "math.ifloor({})",
+    "ceil": "ceil({})",
+    "trunc": "trunc({})",
+    "minimum": "min({}, {})",
+    "maximum": "max({}, {})",
+    "fmod": "fmod({}, {})",
+    "power": "math.pow({}, {})",
+    "float": "dace.float64({})",
+    "float32": "dace.float32({})",
+    "float64": "dace.float64({})",
+    "int": "dace.int32({})" if np.dtype(int).itemsize == 4 else "dace.int64({})",
+    "int32": "dace.int32({})",
+    "int64": "dace.int64({})",
+    "bool": "dace.bool_({})",
+    "plus": "({} + {})",
+    "minus": "({} - {})",
+    "multiplies": "({} * {})",
+    "divides": "({} / {})",
+    "floordiv": "({} // {})",
+    "eq": "({} == {})",
+    "not_eq": "({} != {})",
+    "less": "({} < {})",
+    "less_equal": "({} <= {})",
+    "greater": "({} > {})",
+    "greater_equal": "({} >= {})",
+    "and_": "({} & {})",
+    "or_": "({} | {})",
+    "xor_": "({} ^ {})",
+    "mod": "({} % {})",
+    "not_": "(not {})",  # ~ is not bitwise in numpy
+}
+
+
+@dataclass(frozen=True)
+class GtirTaskletCodegen(codegen.TemplatedGenerator):
+    """
+    Stateless class to visit pure tasklet expressions.
+
+    This visitor class is responsible for building the string representing
+    the Python code inside a tasklet node.
+    """
+
+    def _visit_deref(self, node: itir.FunCall) -> str:
+        assert len(node.args) == 1
+        if isinstance(node.args[0], itir.SymRef):
+            return self.visit(node.args[0])
+        raise NotImplementedError(f"Unexpected deref with arg type '{type(node.args[0])}'.")
+
+    def _visit_numeric_builtin(self, node: itir.FunCall) -> str:
+        assert isinstance(node.fun, itir.SymRef)
+        fmt = _MATH_BUILTINS_MAPPING[str(node.fun.id)]
+        args = self.visit(node.args)
+        return fmt.format(*args)
+
+    def visit_FunCall(self, node: itir.FunCall) -> str:
+        if cpm.is_call_to(node, "deref"):
+            return self._visit_deref(node)
+        elif isinstance(node.fun, itir.SymRef):
+            builtin_name = str(node.fun.id)
+            if builtin_name in _MATH_BUILTINS_MAPPING:
+                return self._visit_numeric_builtin(node)
+            else:
+                raise NotImplementedError(f"'{builtin_name}' not implemented.")
+        raise NotImplementedError(f"Unexpected 'FunCall' node ({node}).")
+
+    @final
+    def visit_Literal(self, node: itir.Literal) -> str:
+        return node.value
+
+    @final
+    def visit_SymRef(self, node: itir.SymRef) -> str:
+        return str(node.id)
```

Comment:
Optional suggestion: this class doesn't seem to use any special functionality of a `TemplatedGenerator` class so it seems it could be written as a regular visitor. An alternative could be to try to use more features  like for example static templates as supported in the `TemplatedGenerator` class
```suggestion
    Literal = as_fmt("{value}")

    SymRef = as_fmt("{id}")
```